src/core/lib/iomgr/cfstream_handle.cc

```diff
@@ -68,8 +71,14 @@ void CFStreamHandle::ReadCallback(CFReadStreamRef stream,
       handle->read_event_.SetReady();
       break;
     case kCFStreamEventErrorOccurred:
-      handle->open_event_.SetReady();
-      handle->read_event_.SetReady();
+      stream_error = CFReadStreamCopyError(stream);
+      error = grpc_error_set_int(
```

Comment:
this error does not seem to get unreffed anywhere. this would be a memory leak

Reply:
thanks for catching this! Will fix.